docs/articles/oso/how-to-open-source-software.rst

```diff
@@ -0,0 +1,28 @@
+:orphan:
+
+How to open source software
+===========================
+
```

Comment:
I'd prefer to call this "How to publish open source software." Otherwise, it seems like the active verb is "open" like you're opening a can of beans. But in many cases, the software is already open, at least the dependencies or OS it runs on since we tend to work on GNU/Linux. Also, many readers are going to be dealing with pushing some code that they may not have written from behind the firewall to the open and I think having the document describe that use case is important and likely the most common.

Reply:
sure

Reply:
I think the title and file name is too broad for what this should be. At least if the intention is to have this being the checklist (of a How-To kind of document), then it should be named more as such, e.g. "Checklist for OSS repo peer review". "How to open source software" is a topic on its own that will include processes, roles, principles, other How-To's etc.

Reply:
I like 'checklist for OSS repo peer review'